src/Sulu/Bundle/AdminBundle/Resources/js/components/Tabs/Tab.js

```diff
@@ -29,17 +87,20 @@ export default class Tab extends React.PureComponent<Props> {
     render() {
         const {
             children,
+            small,
             selected,
         } = this.props;
+
         const tabClass = classNames(
             tabStyles.tab,
             {
-                [tabStyles.selected]: selected,
+                [tabStyles.selected]: this.selected,
+                [tabStyles.small]: small,
             }
         );
 
         return (
-            <li className={tabClass}>
+            <li className={tabClass} ref={this.setListItemRef}>
                 <button
                     disabled={selected}
```

Comment:
This is on purpose `this.props.selected` instead of `this.selected`, because the button should be disabled as soon as the tab is selected